versioned_docs/version-8.2/components/console/manage-organization/delete-account.md

```diff
@@ -4,6 +4,8 @@ title: Delete your Camunda Platform account
 keywords: [account, delete, cancel]
 ---
 
+[//]: <> (Do we retain Camunda Platform in this instance?)
```

Comment:
Can you follow up with @dlevy-camunda on this example? You may need to ping him on Slack.

Reply:
To delete your Camunda account, confirmed 👍